react-components/src/utilities/queryKeys.ts

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type IdEither } from '@cognite/sdk/';
+
+export const queryKeys = {
+  all: ['cdf'] as const,
+  // ASSETS
+  assets: () => [...queryKeys.all, 'assets'] as const,
+  assetsById: (ids: IdEither[]) => [...queryKeys.all, 'assets', ids] as const,
+  // TIMESERIES
+  timeseries: () => [...queryKeys.all, 'timeseries'] as const,
```

Comment:
I wonder if this and the `assets` query keys should be defined outside of this exported object, since they're not supposed to be used outside this file.

Reply:
are you suggesting to remove the assets and timeseries and just define both outside in the places where would be called?

Reply:
No, I just meant putting them outside of the `queryKeys` object, but in this same file, just so that they're not exported/exposed

Reply:
https://github.com/cognitedata/reveal/pull/4414/commits/c9a50e52b105e8f50cceda82d6fe1e4d14b2dfe7